### PR TITLE
Rename badges and bug issue template

### LINKED ISSUE #1281: Readme badges still refers to PowerShellGet
### Summary of the new feature / enhancement

Readme badges still refers to PowerShellGet documentation and gallery link.

https://github.com/PowerShell/PSResourceGet/blob/d59abfe4808b01d884e95e754b95b69bf35d6c84/README.md?plain=1#L3-L4

### Proposed technical implementation details (optional)

_No response_

### PR DESCRIPTION
**Pull request by ThomasNieto**
# PR Summary

Resolves #1281

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [ ] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

**Review by kilasuit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/PowerShell/PowerShellGet/blob/development/LICENSE)
```
Comment (anamnavi) [inline README.md]:
```suggestion
[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/PowerShell/PSResourceGet/blob/master/LICENSE)
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 [![Documentation - PowerShellGet](https://img.shields.io/badge/Documentation-PowerShellGet-blue.svg)](https://docs.microsoft.com/en-us/powershell/module/powershellget/?view=powershell-7.1)
```
Comment (anamnavi) [inline README.md]:
```suggestion
[![Documentation - PSResourceGet](https://img.shields.io/badge/Documentation-PowerShellGet-blue.svg)](https://docs.microsoft.com/en-us/powershell/module/powershellget/?view=powershell-7.1)
```

**Review by anamnavi — CHANGES_REQUESTED** [review]
Thanks for opening this @ThomasNieto, just requested a couple of changes and then we can get this in :)

File: README.md (hunk 2 above)
Comment (ThomasNieto) [reply README.md]: I left that as the underlying documents haven't been renamed yet.

**Review by alerickson — APPROVED** [review]

**Pull request merged**